frontend/src/components/modals/LeaveRoadmapModal.tsx

```diff
@@ -52,6 +55,9 @@ export const LeaveRoadmapModal: Modal<ModalTypes.LEAVE_ROADMAP_MODAL> = ({
       if (res.payload?.message) return setErrorMessage(res.payload.message);
       return;
     }
+    if (currentRoadmapId === roadmapId)
+      dispatch(roadmapsActions.clearCurrentRoadmap());
+    refetchRoadmaps();
```

Comment:
If anyone knows how to use apiV2 in redux actions, refetching roadmaps could be moved to the leaveRoadmap-action. I just could not figure it out.

Reply:
I think this should do: https://redux-toolkit.js.org/rtk-query/usage/usage-without-react-hooks#performing-mutations

Reply:
Interesting; I thought I tried that out. Well, it works now 👍